### PR TITLE
Add Crop Biomass Evalscripts

### PR DESCRIPTION
**Pull request by cpaulik**

### CODE REVIEW

File: planetary-variables/crop-biomass/eob.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+function evaluatePixel(sample) {
+
+let val=sample.CB*factor;
+  [r,g,b]=visualizer.process(val);
+let imgVals = [r,g,b,sample.dataMask];  
+// return[val,val,val,sample.dataMask]  
+    return {
+      default: imgVals,
+      index: [val],     
+      eobrowserStats: [val,sample.dataMask],
```
Comment (jonasViehweger) [inline planetary-variables/crop-biomass/eob.js:41]: Does crop biomass have clouds?

If not, this should change to 

```suggestion
      eobrowserStats: [val],
```

to remove the (unnecessary) cloud slider in EO Browser Stats

File: planetary-variables/crop-biomass/fig/fig1.png
Comment (jonasViehweger) [inline planetary-variables/crop-biomass/fig/fig1.png:1]: Could you retake this screenshot without the text highlighted in the Stats Panel?

**Review by jonasViehweger — COMMENTED** [review]
I'll do some formatting, then with the new screenshot it should be ready

File: planetary-variables/crop-biomass/script.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+//VERSION=3
+function setup() {
+  return {
+    input: ["CB", "dataMask"],
+    output: { bands: 4, sampleTYPE: "AUTO" },
```
Comment (benjaminaouizerats) [inline planetary-variables/crop-biomass/script.js:5]: output: { bands: 4, sampleType: "FLOAT32" },

Comment (jonasViehweger) [reply planetary-variables/crop-biomass/script.js:5]: "AUTO" is fine for visualization. It returns 0-1 scaled to UINT8. Basically AUTO converts reflectances to RGB

**Comment by jonasViehweger** [comment]
Image was updated, everything should be fine. Merging

**Pull request merged**